### PR TITLE
feat(website): Plugins layout

### PR DESCRIPTION
**Pull request by dj-stormtrooper**
#### Summary

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Test locally on your own infrastructure
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation)) 
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

**Review by erezrokah — APPROVED** [review]
🚀

File: website/scripts/generate-hub.js [hunk 1]
```diff
@@ -49,6 +60,14 @@ title: ${NAME_PLACEHOLDER} Plugin Tables
 ${CONTENT_PLACEHOLDER}
 `;
 
+const EXTERNAL_PLUGIN_TEMPLATE = `
+# ${NAME_PLACEHOLDER} Plugin
+
+The CloudQuery ${NAME_PLACEHOLDER} provider pulls configuration out of ${NAME_PLACEHOLDER} resources, normalizes them and stores them in PostgreSQL database.
```
Comment (erezrokah) [inline website/scripts/generate-hub.js:66]:
```suggestion
The CloudQuery ${NAME_PLACEHOLDER} plugin pulls configuration out of ${NAME_PLACEHOLDER} resources, normalizes them and stores them in PostgreSQL database.
```

File: website/scripts/generate-hub.js [hunk 2]
```diff
@@ -12,13 +12,24 @@ const PLUGINS_DATA = {
   k8s: { name: "Kubernetes" },
   okta: { name: "Okta" },
   terraform: { name: "Terraform" },
+  yandexcloud: { name: "Yandex Cloud" },
 };
 
+const EXTERNAL_PLIGINS = [
```
Comment (anton-kuptsov) [inline website/scripts/generate-hub.js:18]: Is it a typo?

Comment (dj-stormtrooper) [reply website/scripts/generate-hub.js]: Thanks!

**Pull request merged**